packages/expo-font/src/Font.ts

```diff
@@ -209,3 +209,4 @@ async function unloadFontInNamespaceAsync(
 }
 
 export { FontDisplay, FontSource, FontResource, UnloadFontOptions };
+export { useFonts } from './FontHook';
```

Comment:
How about renaming `FontHook` to `FontHooks`? It will be future-proof! (That's what we do in `expo-auth-session`. I've also seen people use the structure like `src/hooks/use-fonts.ts`. I don't think there is a strong preference for either the former or the latter, apart from trying to be consistent across the codebase.)

Reply:
I think `FontHooks` is the best one for now, it matches the current API files better than `src/hooks` I think 😄